### ISSUE TITLE
[0.8] HTMLAnchor has a `host` property, breaks the intended behavior of `Polymer.Base._queryHost`

### ISSUE THREAD

**Issue by cdata** [Polymer/polymer#1359]
`Polymer.Base._queryHost` attempts to resolve a `host` node by walking through DOM ancestors and measuring their `host` properties. Unfortunately, HTMLAnchor has a built-in `host` property that is the hostname component of its `href` property, if available. This has the unfortunate side-effect of causing a runtime exception if a Polymer custom element is wrapped with an HTMLAnchor in ShadeyDOM mode.

**Comment by IntranetFactory** [comment]
I got the following error in ready.html, line 87

`Uncaught TypeError: Cannot read property 'push' of undefined`

I think that's the same problem as mentioned above. My workaround was to ignore HTMLAnchorElement by changing _queryHost in ready.html as follows:

```
   _queryHost: function(node) {
      return node && 
        ((node.host && node.constructor.name != "HTMLAnchorElement") || (node.host = this._queryHost(Polymer.dom(node).parentNode)));
    },
```

**Comment by IntranetFactory** [comment]
I think that https://github.com/Polymer/polymer/pull/1372 solved this issue, at least for me.

**Comment by sorvell** [comment]
Fixed in version 0.9.
